workflow/rules/bwa_mem.smk

```diff
@@ -0,0 +1,51 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule bwa_mem:
+    input:
+        reads=["pre-alignment/merge/{sample}_{unit}_R1.fq.gz", "Pre-alignment/merge/{sample}_{unit}_R2.fq.gz"],
+        index=config["references"]["genome"],
+    output:
+        bam="alignment/bwa_mem/{sample}_{unit}.bam",
+    params:
+        index=config["references"]["genome"],
+        extra=config["bwa_mem"]["extra"],
+        sort=config["bwa_mem"].get("sort", "samtools"),
+        sort_order=config["bwa_mem"].get("sort_order", "coordinate"),
+        sort_extra="-@ " + str(config["bwa_mem"]["threads"]),
+    log:
+            log="aligment/bwa_mem/{sample}_{unit}.bam.log",
```

Comment:
I think log does not need a field name `log=`, we probably do not want different log files for the same rule. And the indentation is one tab too many.

Reply:
done